client/apps/plugin-status/indicator.js

```diff
@@ -29,21 +29,11 @@ const Indicator = ( { title, subtitle, state, message, children } ) => {
 			icon = 'notice';
 	}
 
-	const renderSubTitle = () => {
```

Comment:
No need to create a new function at each re-render just so that the function is called immediately